dev/build.image/build.gradle

```diff
@@ -383,6 +388,36 @@ if (isAutomatedBuild && !isIFIXBuild) {
             }
         }
     }
+    
+    
+    task packageOpenLibertyWebProfile9(type: PackageLibertyWithFeatures) {
+        doFirst {
```

Comment:
Do we need to exclude ee 8 features as well?

Reply:
No ... I'm not even convinced that these existing ee7 exclusions are still needed